README.md

```diff
@@ -14,15 +14,23 @@ The main resource developed in our paper was the "Improving Question Naturalness
 
 This repository is structured as follows:
 
-- `test_collection/`: The files that constitute the test collection IQN-KGQA.
+- `test_collection/`: Contains the files that constitute the test collection IQN-KGQA.
 - `test_collection/final_questions.csv`: Original and final NL questions in test collection. This CSV file has four columns: `Dataset`, `QID`, `OriginalQuestion`, and `FinalQuestion`. 
+- `test_collection/DBNQA\*/`: Contains the files corresponding the NL questions sampled from DBNQA*. 
+- `test_collection/DBNQA\*/test_sample_original.en`: The original NL questions sampled from DBNQA* test split. 
+- `test_collection/DBNQA\*/test_sample_final.en`: The final NL questions after crowdsourced rewrites were substituted. 
+- `test_collection/DBNQA\*/test_sample.sparql`: The SPARQL queries corresponding line-by-line to either of the two preceding NL questions files.
+- `test_collection/GrailQA_v1.0/grailqa_v1.0_dev_sample_original.json`: The original instances (including NL questions) sampled from GrailQA dev split.
+- `test_collection/GrailQA_v1.0/grailqa_v1.0_dev_sample_final.json`: The final instances with NL questions substituted by crowdsourced rewrites.
+- `test_collection/LC_QuAD_2.0/test_sample_original.json`: The original instances (including NL questions) sampled from LC-QuAD v2.0 test split.
+- `test_collection/LC_QuAD_2.0/test_sample_final.json`: The final instances with NL questions substituted by crowdsourced rewrites.
```

Comment:
instead of 
```
- test_collection/xxx...
``` 
use an extra level of indentation and drop `test_collection`, e.g., 
```
    * xxx....
 ```

Reply:
Done.